app/resources/add_note_base.html.template

```diff
@@ -195,45 +195,46 @@
           <div class="end-multi-columns"></div>
         {% endif %}
       </div>
-
-      {% if env.enable_photo_upload %}
-        <div class="photo subsection">
-          <h3>Photo</h3>
-          <div class="hint">
-            {% trans "To attach a photo to this note, upload it or enter its URL." %}
-          </div>
-          <div class="field" onclick="$('note_photo_url_radio').checked=true; update_image_input(true);">
-            <span class="radio">
-                <input id="note_photo_url_radio" type="radio"
-                    checked="checked" name="note_photo_input">
-            </span>
-            <span class="label">
-              {% trans "URL" %}:
-            </span>
-            <span class="value">
-              <input name="note_photo_url" id="note_photo_url"
-                  class="medium-text-input"
-                  value="{{params.photo_url}}" />
-            </span>
-          </div>
-          <div class="field" onclick="$('note_photo_upload_radio').checked=true; update_image_input(true);">
-            {% comment %} This td can be removed if uploading needs to be disabled {% endcomment %}
-            <span class="radio">
-              <input id="note_photo_upload_radio" type="radio"
-                  name="note_photo_input">
-            </span>
-            <span class="label">
-              {% trans "Upload" %}:
-            </span>
-            <span class="value">
-              <input class="photo-upload-input" id="note_photo_upload" type="file"
-                  name="note_photo" disabled="disabled" onchange="show_photo_remove_icon('note_photo_upload')">
-              <a href="javascript:remove_photo_upload('note_photo_upload')" id="remove_note_photo_upload" style="display: none"><img class="photo-upload-cancel" src="{{env.global_url}}/cancel.png" alt="Cancel"></a>
-            </span>
-            {% comment %} see above comment {% endcomment %}
+      {% if not params.create_mode %}
```

Comment:
Can you add a comment why you hide this if it's create mode?

Hint: You can use {# ... #} for comments in Django template.